DeAceroDLabs/marketplace-frontend
# Feature/dmp 604 import google maps api

## What is done

[[DMP-604] import google maps api](https://deacero.atlassian.net/browse/DMP-604)

## How is done

- Created a `Map` component that will be used everytime you need a google map.
- Created a `MapField` to use it in signup form later here we will send the lat and lng.
- Added REACT_APP_GOOGLE_MAPS_API_KEY on .env file locally and in the vm. (ask Diana for the value)

## How to test

1. Run the project via `npm start`
2. Click on Registrarse.
3. Fill the first form.
4. In the second form you should see the map.
